docs/configuration/index-config.md

```diff
@@ -321,7 +328,7 @@ expand_dots: false
 | `description` | Optional description for the field. | `None` |
 | `stored`    | Whether value is stored in the document store | `true` |
 | `indexed`   | Whether value is indexed | `true` |
-| `fast`   | Whether value is fast | `false` |
+| `fast`     | Whether value is stored in a fast field. The default behaviour for text is to store the original text unchanged. It can be configured via `{"normalizer": "lowercase"}`. ([See normalizers](#description-of-available-normalizers)) for a list of available normalizers. | `true` |
```

Comment:
can you update the mapping example above too?
I would also replace `{"normalizer": "lowercase"}` by `normalizer: lowercase` as we use YAML in the docs.

Reply:
Update both